packages/components/src/components/vertical-tab/vertical-tab.jsx

```diff
@@ -28,17 +28,17 @@ const VerticalTab = ({
     const [curr_tab_index, setCurrTabIndex] = React.useState(vertical_tab_index || 0);
     const [selected_element, setSelectedElement] = React.useState(list[0]);
 
-    const setSelectedIndex = ({ list, selected_index, is_routed, current_path }) => {
```

Comment:
circleci is throwing an error `these variables are already declared in the upper scope `. there are two solutions to this make this function use the outer variable by omitting these variables from the object parameter of the inner function. or re-name the object keys.